src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs

```diff
@@ -1490,5 +1490,7 @@ internal DataTemplate ResolveItemTemplate(object item)
 		/// Resets internal cached state of the collection.
 		/// </summary>
 		private protected virtual void Refresh() { }
+
+		public Panel ItemsHost { get; }
```

Comment:
Is this set somewhere?? And it should be internal, it's not part of the public contract.

Reply:
Removed and it was replaced by the `ItemsPanelRoot` as they represent the same object.